src/ccontrol/MergingHandler.h

```diff
@@ -60,6 +60,9 @@ namespace lsst::qserv::ccontrol {
 class MergingHandler : public qdisp::ResponseHandler {
 public:
     typedef std::shared_ptr<MergingHandler> Ptr;
+
+    enum MergeState { PREMERGE, MERGING, CANCELLED };
```

Comment:
(Just curious, since I don't fully understand the logic glow if the result merges in this version of Qserv) - Should there be any state indicating successful completion of the merge?